config.sample.json

```diff
@@ -0,0 +1,45 @@
+{
+    "coinCapUri": "https://api.coincap.io/v2/assets/",
```

Comment:
Use the "rates1.edge.app" server instead of coincap

Reply:
Updated per comment. Replaced `coinCapUri` with `ratesServerAddress`.